Index: trousers-0.3.13/src/tcs/ps/ps_utils.c
===================================================================
--- trousers-0.3.13/src/tcs/ps/ps_utils.c
+++ trousers-0.3.13/src/tcs/ps/ps_utils.c	2015-10-23 17:30:43.597548033 +0000
@@ -45,7 +45,7 @@
 #ifdef SOLARIS
 TSS_RESULT
 #else
-inline TSS_RESULT
+TSS_RESULT
 #endif
 read_data(int fd, void *data, UINT32 size)
 {
@@ -67,7 +67,7 @@
 #ifdef SOLARIS
 TSS_RESULT
 #else
-inline TSS_RESULT
+TSS_RESULT
 #endif
 write_data(int fd, void *data, UINT32 size)
 {
Index: trousers-0.3.13/src/tspi/ps/ps_utils.c
===================================================================
--- trousers-0.3.13/src/tspi/ps/ps_utils.c
+++ trousers-0.3.13/src/tspi/ps/ps_utils.c	2015-10-23 17:33:41.963794424 +0000
@@ -22,7 +22,7 @@
 #include "tspps.h"
 #include "tsplog.h"
 
-inline TSS_RESULT
+TSS_RESULT
 read_data(int fd, void *data, UINT32 size)
 {
 	int rc;
@@ -39,7 +39,7 @@
 	return TSS_SUCCESS;
 }
 
-inline TSS_RESULT
+TSS_RESULT
 write_data(int fd, void *data, UINT32 size)
 {
 	int rc;
