diff -p -up trousers-0.3.1/dist/fedora/fedora.initrd.tcsd.lsb trousers-0.3.1/dist/fedora/fedora.initrd.tcsd
--- trousers-0.3.1/dist/fedora/fedora.initrd.tcsd.lsb	2009-08-06 13:32:54.000000000 -0300
+++ trousers-0.3.1/dist/fedora/fedora.initrd.tcsd	2009-08-06 13:35:41.000000000 -0300
@@ -8,6 +8,12 @@
 # processname: tcsd
 # config: /etc/tcsd.conf
 # pidfile: /var/run/tcsd.pid
+#
+### BEGIN INIT INFO
+# Provides: tcsd
+# Short-Description: trusted computing daemon
+# Description: TrouSerS server daemon acts as a gateway to TPM device driver
+### END INIT INFO
 
 # source function library
 . /etc/rc.d/init.d/functions
